Alpaca.Markets/Interfaces/IHistoricalTradeV2.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Alpaca.Markets
+{
+    /// <summary>
+    /// Encapsulates historical trade information from Polygon REST API.
+    /// </summary>
+    public interface IHistoricalTradeV2
+    {
+        /// <summary>
+        /// Gets SIP timestamp
+        /// </summary>
+        Int64 SipTimestamp { get; }
```

Comment:
We have a lot of similar interface items which can be extracted into a separate interface for better re-use.